server/tests/test_mpesa_routes.py

```
@@ -0,0 +1,277 @@
+            'CustomerMessage': 'Success. Request accepted for processing'
+        }
+
+        data = {
+            'amount': 100,
+            'phone_number': '254712345678',
+            'invoice_id': 1,
+            'description': 'Test payment'
+        }
+
+        response = client.post('/api/mpesa/stk-push', 
+                             data=json.dumps(data),
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)